eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/ExcludeStructuralVariantsProcessor.java

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.pipeline.steps.processors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.ItemProcessor;
+
+import uk.ac.ebi.eva.commons.core.models.IVariant;
+
+import java.text.MessageFormat;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ *  Direct implementation of VCF specification grammars from
+ *  <a href="https://github.com/EBIvariation/vcf-validator/blob/master/src/vcf/vcf.ragel">here</a> and
+ *  <a href="https://github.com/EBIvariation/vcf-validator/blob/master/src/vcf/vcf_v43.ragel">here</a>.
+ */
+public class ExcludeStructuralVariantsProcessor implements ItemProcessor<IVariant, IVariant> {
+
+    private static final Logger logger = LoggerFactory.getLogger(ExcludeStructuralVariantsProcessor.class);
+
+    private static String meta_contig_char = "(\\p{Alnum}|[\\p{Punct}&&[^:<>\\[\\]*=,]])";

```

Comment:
`alnum` and `punct` are Ragel-specific terms. For more standard regexp you probably need to use `a-zA-Z0-9` or similar.

Reply:
Alnum and Punct are also character classes available in the Java regex engine. See https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html